.setup/pip/system_requirements.txt

```diff
@@ -42,7 +42,7 @@ PyPDF2==1.26.0
 distro==1.6.0
 jsonschema==3.2.0
 jsonref==0.2
-docker==5.0.1
+docker==5.0.0
```

Comment:
intentional change?

Reply:
Nope, not intentional.  Will revert.  I suspect this may come from when I merged master in and then reverted the change.